---
title: Research in the Omar Lab
layout: default
group: research
---

<div class="row">

# Our research
We focus on developing robust artificial intelligence (AI)-powered multimodal risk-stratification tools to forecast the risk of disease progression in cancer patients. These tools incorporate the composition of the tumor microenvironment (TME) in their training to improve their robustness, accuracy, and generalizability to different patient cohorts.
#### We work with different data modalities, including genomics, histopathology, and radiology images to build robust AI models for risk assessment in patients newly diagnosed with cancer. A key component of our research is using the composition of the tumor microenvironment to extract high-quality biological features that serve as supervisors for the training process, guarding against overfitting.
<br>
<br>

</div>

<div class="row">

### Identifying integrated pathomic and transcriptomic signatures of high-risk cancer phenotypes

<div class="col-md-7 order-md-1">

Traditional unimodal analyses fall short in capturing the systemic complexity of the TME dynamics driving cancer invasiveness and metastasis. This limitation significantly hinders our ability to design effective risk-stratification approaches that can inform clinical management. The integration of features derived from multiple biomedical data modalities offers a comprehensive approach to characterize the tumor dynamics on a microscopic scale (see [Omar et al. 2024](https://www.annualreviews.org/content/journals/10.1146/annurev-cancerbio-062822-010523)) and to discern multimodal features high-risk disease at early stages. 
Our lab focuses on developing stage-specific tissue modules that comprehensively capture the complex composition and interplay within the TME at early stages preceding progression. These modules comprise cellular composition, expression profiles, intercellular interactions, gene regulatory networks, and spatial neighborhoods. This approach provides an in-depth, systemic understanding of the TME, highlighting the diversity of cell types and the extensive network of molecular and spatial interactions that dictate cellular function and the trajectory of progression. 
We employ high-resolution spatial omics to generate single cell-resolution data that recapitulates the spatial distribution and abundance of mRNAs and specific proteins within the tissue context. We leverage this data to construct detailed maps of cellular composition, molecular expression and spatial localization, facilitating a nuanced analysis of how these factors interact within the physical confines of the TME as the tumor progresses. 

</div>
<div class="col-md-5 order-md-2 align-self-center">
<img class="img-fluid" src="/static/img/pub/digitalpath.gif" alt="qFit">
<!-- <a href="http://www.ucsf.edu"><img class="inline-block navb-icon" src="/static/img/ucsf_logo_white.svg" alt="University of California, San Francisco (UCSF) logo"></a> -->

</div>
</div>
<div class="row">

### Using multi-temperature X-ray data collection methods in mechanistic studies and ligand discovery campaigns

<div class="col-md-7 order-md-2">

We recognized that the standard practice of cryocooling crystals could distort protein conformations.
In both larger surveys and isolated mechanistic studies, we have demonstrated the value of room temperature data collection for revealing the structural basis of protein conformational dynamics, leading to new insights into enzymes, and increasing connections to dynamics studies from NMR and simulations.
Additionally, we have identified how temperature can bias small molecule discovery, leading some fragment sites inaccessible at cryogenic temperatures and repositioning critical water molecules.
</div>

<div class="col-md-5 order-md-1 align-self-center">
<img class="img-fluid" src="/static/img/pub/2018_keedy_hill.jpg" alt="ptp1b">
</div>
</div>
<div class="row">

### Determining structures that influence microbial-host interactions

<div class="col-md-7 order-md-1 ">

We are interested in the structural enzymology and potential therapeutic applications of the human enzymes that degrade chitin, the pervasive polymer that can cause inflammation in the context of allergy and asthma. Using high resolution cryo-electron microscopy (cryo-EM), we are studying the mechanisms of action to newly synthesized antibiotics (in collaboration with the [Seiple](https://seiplegroup.ucsf.edu/) and [Fujimori](https://fujimorilab.ucsf.edu) labs). With this platform, we are also pushing on cryo-EM data processing and refinement methods to enable structure-based drug design, now achieving resolutions better than 2.0 Ã…. With the improved molecules we generate, we are defining potential resistance mechanisms to these antibiotics using a combination of structural biology and deep mutational scanning experiments. In recent work, we are collaborating with the Bondy-Denomy lab to define the structural basis of anti-CRISPR proteins using a combination of X-ray crystallography, cryo-EM, and NMR. The mechanism of action for many of these proteins remains mysterious.
</div>

<div class="col-md-3 order-md-2 align-self-center">

<img class="img-fluid" src="/static/img/pub/2019_li_pellegrino.jpg" alt="antibiotic">
</div>
</div>
<div class="row">

### Identifying unifying concepts between systems and structural biology

<div class="col-md-7 order-md-2">


We are intrigued by the similarities in genetic epistasis and thermodynamic measurements and are applying these insights to large-scale studies of point mutants and posttranslational modifications. We attempt to connect the response to genetic (mutation), chemical (ligands), and physical (temperature) in experiments to define the basis of allosteric regulation and function. We like to bring an evolutionary biology perspective to the table to understand the relative importance of these different perturbations in shaping the proteins we see today.
</div>

<div class="col-md-5 order-md-1 align-self-center">
<img class="img-fluid" src="/static/img/pub/2020_gordon_jang_bouhaddou_xu_obernier_white_omeara_rezelj.png" alt="hairball">
</div>
</div>
<div class="row">

### Creating new time-resolved experiments to probe correlated motions in proteins

<div class="col-md-7 order-md-1">

A major limitation of most biophysical techniques is the inability to directly reveal correlations in motions between distinct regions of macromolecules.
We are taking advantage of the new capabilities of next-generation X-ray free electron laser (X-FEL) light sources to perform radiation damage-free imaging of proteins and to watch how protein ensembles respond when perturbed by rapid temperature jumps using the X-FEL.
At equilibrium, X-ray diffuse scattering has the potential to reveal these motions; however, we currently lack the ability to collect, integrate, and refine diffuse scattering data.
Our long-term goal is to increase the information content of every X-ray diffraction experiment to reveal atomic level coupling at high resolution and improved models of grouped flexibility at low resolution.
</div>

<div class="col-md-5 order-md-2 align-self-center ">
<img class="img-fluid" src="/static/img/pub/2019_thompson.jpg" alt="tjump">
</div>

</div>
